features/steps/wishlist_steps.py

```diff
@@ -22,3 +22,167 @@
 WISHLIST_PREFIX = 'wishlist_'
 ITEM_PREFIX = 'item_'
 
+@given('the following wishlists')
+def step_impl(context):
+    """ Delete all Wishlists and load new ones """
+    headers = {'Content-Type': 'application/json'}
+    # list all of the wishlists and delete them one by one
+    context.resp = requests.get(context.base_url + '/wishlists', headers=headers)
+    expect(context.resp.status_code).to_equal(200)
+    for wishlist in context.resp.json():
+        context.resp = requests.delete(context.base_url + '/wishlists/' + str(wishlist["id"]), headers=headers)
+        expect(context.resp.status_code).to_equal(204)
+    
+    # load the database with new wishlists
+    create_url = context.base_url + '/wishlists'
+    
+    for row in context.table:
+        data = {
+            "name": row['name'],
+            "user_id": row['user_id'],
+            "items": row['items'],
```

Comment:
I am not sure if items is a valid column here in the wishlist table.

Reply:
Yes, I use empty item list for new wishlist. As shown in the "behave", it could pass the test. 
I found you use [laptop] in the items column in .feature file, that is not valid. Because item contains more than item name, and also row['items'] pass a String variable

Reply:
As you mentioned, the column itself is not valid. Let's remove the column from the file.

Reply:
Maybe put an empty list in the request in the js when creating a wishlist? Putting it in the feature file is not right as you mentioned, the item as a lot more properties than name.

Reply:
Done.